pkgs/by-name/libresoc-nmigen/lib.nix

```diff
@@ -0,0 +1,12 @@
+{fetchgit}: {
+  fetchFromLibresoc = {
+    pname,
+    hash,
+    rev,
+    fetchSubmodules ? true,
+  }:
+    fetchgit {
```

Comment:
nice suggestion from Lorenz to consolidate into this function to remove the duplicate comments, but shouldn't you at least have one instance of the comment here? Otherwise I read this and wonder why it's even needed instead of using fetchFromGitHub